sonar-project.properties

```diff
@@ -0,0 +1,4 @@
+sonar.projectKey=Open-MBEE_flexo-mms-auth-service
```

Comment:
Are these necessary when they're already set in the build.gradle.kts?

Reply:
They are, the orb looks at this file for it's settings. The CI does not actually run the gradle command, I left the gradle settings just as convenience for running the sonar analysis locally without needing to commit. I tried looking at ways to use properties from other properties files in `build.gradle.kts` but haven't found best practices for that yet, if you know of a way please advise.

The reason for using the Orb instead of the gradle command was because based on my testing PRs are not automatically linked when running the gradle scan command, you would have to provide [this additional information](https://docs.sonarsource.com/sonarcloud/advanced-setup/ci-based-analysis/other-cis/).